types/leaflet.sync/index.d.ts

```diff
@@ -0,0 +1,38 @@
+// Type definitions for Leaflet.Sync 0.2
+// Project: https://github.com/jieter/Leaflet.Sync
+// Definitions by: Gabriel O. Martins <https://github.com/OliverBiel/>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+// Minimum TypeScript Version: 5.0
+
+import * as L from 'leaflet';
+
+declare module 'leaflet' {
+    interface SyncMapOptions {
+        noInitialSync?: boolean;
+        syncCursor?: boolean;
+        syncCursorMarkerOptions?: CircleMarkerOptions;
+        offsetFn?: (center: LatLngExpression, zoom: number, refMap: Map, targetMap: Map) => LatLngExpression;
+    }
+
+    interface Map {
+        sync(map: Map, options?: SyncMapOptions): this;
+        unsync(map: Map): this;
+        isSynced(otherMap?: Map): boolean;
+        _cursorSyncMove(e: LeafletMouseEvent): void;
+        _cursorSyncOut(e: LeafletMouseEvent): void;
+        _selfSetView(e: LeafletEvent): void;
+        _syncOnMoveend(e: LeafletEvent): void;
+        _syncOnDragend(e: LeafletEvent): void;
+        _initSync(): void;
```

Comment:
```suggestion
```
These seem to be not part of API so look like internal implementation details. Are you sure you need them